src/index.js

```diff
@@ -83,6 +84,8 @@ function ParseServer({
   enableAnonymousUsers = true,
   oauth = {},
   serverURL = '',
+  verifyUserEmails = false,
```

Comment:
I named it this way for consistency with with the dashboard which uses "Verify user emails". I could go for `enableUserEmailVerification` to be consistent with `enableAnonymousUsers` though.